### PR TITLE
fix exercise 2.29 (e)

### PR DESCRIPTION
**Pull request by ifgladlee**
fix exercise 2.29 (e)

### CODE REVIEW

**Comment by ifgladlee** [comment]
fix exercise 2.29 (e)

File: ch02/README.md [hunk 1]
```diff
@@ -437,7 +437,7 @@ i = ic;     // legal.
 p1 = p3;    // illegal. p3 is a pointer to const int.
 p1 = &ic;   // illegal. ic is a const int.
 p3 = &ic;   // legal.
-p2 = p1;    // legal.
```
Comment (pezy) [inline ch02/README.md]: p2 hasn't initialized yet.

Comment (ifgladlee) [inline ch02/README.md]: p2 hasn't been initialized, which is an error in `Exercise 2.28` .when we do `Exercise 2.29`,we have to assume `p2` had been initialized correctly, right? After all, it's not allowed to assign a value to a const pointer.

Comment (pezy) [inline ch02/README.md]: yes, you are right.

If we assume `p1`, `p2`, `p3` all have been initialized. `p3 = &ic` is also an error, cause `p3` is `const int *const` type. right?

Comment (ifgladlee) [inline ch02/README.md]: Right. I've missed it. `p3 = &ic` is also illegal.I'll fix it.

**Comment by pezy** [comment]
nice ! :+1:

**Pull request merged**